[editor]/editor_main/server/mapEditorScriptingExtension_s.lua

```diff
@@ -1,6 +1,6 @@
--- FILE: 	mapEditorScriptingExtension_s.lua
--- PURPOSE:	Prevent the map editor feature set being limited by what MTA can load from a map file by adding a script file to maps
--- VERSION:	RemoveWorldObjects (v1) AutoLOD (v1)
+-- FILE: mapEditorScriptingExtension_s.lua
+-- PURPOSE: Prevent the map editor feature set being limited by what MTA can load from a map file by adding a script file to maps
+-- VERSION: RemoveWorldObjects (v1) AutoLOD (v1)

```

Comment:
Probably worth bumping the version while at it, since something is being changed.
```suggestion
-- VERSION: RemoveWorldObjects (v1) AutoLOD (v2)

```